### PR TITLE
feature(gce): support new region: us-west1

### PR DESCRIPTION
**Pull request by amoskong**
https://trello.com/c/rxP8drlL/3501-support-1-more-region-in-gce-with-new-gce-runner

New sct builder had been created and registered to Jenkins.

This patch added support for artifact-test and upgrade-test to
use new GCE region.

Signed-off-by: Amos Kong <amos@scylladb.com>

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by amoskong** [comment]
byo test: https://jenkins.scylladb.com/job/scylla-master/job/reproducers/job/byo-artifact-centos7-test/6/console

File: sdcm/utils/gce_utils.py [hunk 1]
```diff
@@ -30,11 +30,14 @@
 
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
+    assert region.startswith("us-east1") or region.startswith("us-west1"), "only supports two regions"
+    if region.count("-") != 1:
+        return region
+    if region.startswith("us-east1"):
```
Comment (dkropachev) [inline sdcm/utils/gce_utils.py:36]:
```suggestion
    if region == "us-east1":
```

File: sdcm/utils/gce_utils.py [hunk 2]
```diff
@@ -30,11 +30,14 @@
...
         # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
         return f"{region}-{random.choice('bccdd')}"  # choose zones c and d twice as often as zone b
-    return region
+    elif region.startswith("us-west1"):
```
Comment (dkropachev) [inline sdcm/utils/gce_utils.py:39]:
```suggestion
    if region == "us-west1":
```

File: vars/getJenkinsLabels.groovy [hunk 3]
```diff
@@ -38,6 +40,26 @@ def call(String backend, String aws_region=null) {
...
+            datacenter = gce_supported_datacenters[0]
+        }
+        def cp_datacenter = cloud_provider + "-" + datacenter
+        println("Checking if we have a label for " + cp_datacenter)
+        def label = jenkins_labels.get(cp_datacenter, null)
+        if (label != null){
+            println("Found GCE builder with label: " + label)
+            return [ "label": label, "region": datacenter ]
+        }
+        else{
```
Comment (dkropachev) [inline vars/getJenkinsLabels.groovy:59]:
```suggestion
        } else {
```

File: vars/getJenkinsLabels.groovy [hunk 4]
```diff
@@ -38,6 +40,26 @@ def call(String backend, String aws_region=null) {
             throw new Exception("=================== AWS region ${aws_region} not supported ! ===================")
```
Comment (dkropachev) [inline vars/getJenkinsLabels.groovy:40]:
```suggestion
    } else {
```

File: vars/artifactsPipeline.groovy [hunk 5]
```diff
@@ -118,6 +121,7 @@ def call(Map pipelineParams) {
                                                         export SCT_REGION_NAME="${params.region_name}"
                                                     elif [[ ! -z "${params.gce_image_db}" ]]; then
                                                         export SCT_GCE_IMAGE_DB="${params.gce_image_db}"
+                                                        export SCT_GCE_DATACENTER="${params.gce_datacenter}"
```
Comment (dkropachev) [inline vars/artifactsPipeline.groovy:124]: Please add it in the following manner:

```
if [[ -n "${params.gce_datacenter ? params.gce_datacenter : ''}" ]] ; then
    export SCT_GCE_DATACENTER=${params.gce_datacenter}
fi
```

File: vars/rollingUpgradePipeline.groovy [hunk 6]
```diff
@@ -100,8 +103,11 @@ def call(Map pipelineParams) {
 
                                                         export SCT_CLUSTER_BACKEND=${params.backend}
 
-                                                        if [[ -n "${params.aws_region ? params.aws_region : ''}" ]] ; then
-                                                            export SCT_REGION_NAME=${aws_region}
+                                                        if [[ -n "${params. ? params.aws_region : ''}" ]] ; then
```
Comment (dkropachev) [inline vars/rollingUpgradePipeline.groovy:106]:
```
if [[ -n "${params.aws_region ? params.aws_region : ''}" ]] ; then
    export SCT_REGION_NAME=${params.aws_region}
fi
```

File: vars/rollingUpgradePipeline.groovy [hunk 7]
```diff
@@ -100,8 +103,11 @@ def call(Map pipelineParams) {
...
+                                                            export SCT_REGION_NAME=${params.aws_region}
+                                                        fi
+                                                        if [[ -n "${params. ? params.gce_datacenter : ''}" ]] ; then
```
Comment (dkropachev) [inline vars/rollingUpgradePipeline.groovy:109]:
```
if [[ -n "${params.gce_datacenter ? params.gce_datacenter : ''}" ]] ; then
    export SCT_GCE_DATACENTER=${params.gce_datacenter}
fi

```

**Review by dkropachev — CHANGES_REQUESTED** [review]
Please also fix following pipelines:

```
./vars/byoOperatorPipeline.groovy
./vars/perfRegressionParallelPipeline.groovy
./vars/rollingOperatorUpgradePipeline.groovy
./vars/managerPipeline.groovy
./vars/longevityPipeline.groovy
./vars/jepsenPipeline.groovy
./vars/byoLongevityPipeline.groovy
./vars/cdcReplicationPipeline.groovy
```

File: sdcm/utils/gce_utils.py (hunk 1 above)
Comment (roydahan) [reply sdcm/utils/gce_utils.py:36]: Maybe because in GCE the region include the zone?
Like "us-east1b"

File: sdcm/utils/gce_utils.py [hunk 8]
```diff
@@ -30,11 +30,14 @@
...
+        return f"{region}-{random.choice('abc')}"
```
Comment (roydahan) [inline sdcm/utils/gce_utils.py:40]: Don't there are more than a, b and c?

File: sdcm/utils/gce_utils.py [hunk 9]
```diff
@@ -30,11 +30,14 @@
 
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
+    assert region.startswith("us-east1") or region.startswith("us-west1"), "only supports two regions"
```
Comment (roydahan) [inline sdcm/utils/gce_utils.py:33]: I think it's better to prepare from now to the option we will have list of supported regions not just 2.

File: sdcm/utils/gce_utils.py (hunk 1 above)
Comment (amoskong) [reply sdcm/utils/gce_utils.py:36]: > Maybe because in GCE the region include the zone?
> Like "us-east1-b" ?

Yes, we didn't prevent to include available zone suffix in gce_datacenter.

```
defaults/gce_config.yaml:gce_datacenter: 'us-east1'
defaults/k8s_gce_minikube_config.yaml:gce_datacenter: 'us-east1'
defaults/k8s_gke_config.yaml:gce_datacenter: 'us-east1-b'
docs/configuration_options.md:| **<a href="#user-content-gce_datacenter" name="gce_datacenter">gce_datacenter</a>**  | Supported: us-east1 - means that the zone will be selected automatically or you can mention the zone explicitly, for example: us-east1-b | N/A | SCT_GCE_DATACENTER
internal_test_data/complex_test_case_with_version.yaml:gce_datacenter: 'us-east1-b'
```

File: sdcm/utils/gce_utils.py (hunk 9 above)
Comment (amoskong) [reply sdcm/utils/gce_utils.py:33]: The only thing we need to do for support more gce region is creating sct builder in the region by an image:
- https://console.cloud.google.com/compute/machineImages/details/sct-builder-image-20210716-v1?project=skilled-adapter-452

I will add
GCE_SUPPORTED_REGIONS = ["us-east1", "us-west1"]

File: sdcm/utils/gce_utils.py (hunk 8 above)
Comment (amoskong) [reply sdcm/utils/gce_utils.py:40]: There are only 3 available zones for us-west1
- https://cloud.google.com/compute/docs/regions-zones

File: vars/artifactsPipeline.groovy (hunk 5 above)
Comment (amoskong) [reply vars/artifactsPipeline.groovy:124]: OK

File: vars/rollingUpgradePipeline.groovy (hunk 7 above)
Comment (amoskong) [reply vars/rollingUpgradePipeline.groovy:109]: fixed.

File: vars/rollingUpgradePipeline.groovy (hunk 6 above)
Comment (amoskong) [reply vars/rollingUpgradePipeline.groovy:106]: fixed.

File: vars/getJenkinsLabels.groovy (hunk 4 above)
Comment (amoskong) [reply vars/getJenkinsLabels.groovy:40]: fixed. Thanks

File: sdcm/utils/gce_utils.py [hunk 10]
```diff
@@ -28,13 +28,21 @@
...
+
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
-        # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
-        return f"{region}-{random.choice('bccdd')}"  # choose zones c and d twice as often as zone b
-    return region
+    for support_region, zone_string in SUPPORT_REGIONS.items():
+        if region == support_region:
```
Comment (dkropachev) [inline sdcm/utils/gce_utils.py:41]: @amoskong , isn't it suppose to be
```suggestion
        if region.startswith(support_region)
```

Comment (dkropachev) [reply sdcm/utils/gce_utils.py:41]: Also one more question ? do we need to validate region_name+datacenter cases ? if that is the case we need to redo entire logic here

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: vars/getJenkinsLabels.groovy [hunk 11]
```diff
@@ -14,6 +14,8 @@ def call(String backend, String region=null) {
                           'aws-eu-north-1': 'aws-sct-builders-eu-north-1',
                           'aws-eu-central-1': 'aws-sct-builders-eu-central-1',
                           'aws-us-east-1' : 'aws-sct-builders-us-east-1-new',
+                          'gce-us-east1': 'gce-sct-builders-us-east1',
+                          'gce-us-west1': 'gce-sct-builders-us-west1',
                           'gce': 'gce-sct-builders',
```
Comment (aleksbykov) [inline vars/getJenkinsLabels.groovy:19]: Do we need this option?

File: sdcm/utils/gce_utils.py [hunk 12]
```diff
@@ -28,13 +28,21 @@
...
+            if region.count("-") != 1:
+                return region
+            return f"{region}-{random.choice(zone_string)}"
+    raise Exception(f'Unsupported region: {region}')
```
Comment (aleksbykov) [inline sdcm/utils/gce_utils.py:45]: If exception is raised, then we need another PR to scylla-pkg, which will switch the jobs which runs on gce to use only those 2 datacenters by default. Otherwise, the job continue to failed due to wrong region.

File: sdcm/utils/gce_utils.py (hunk 10 above)
Comment (aleksbykov) [reply sdcm/utils/gce_utils.py:41]: i think, in this case, the region and dc is same.

**Review by aleksbykov — COMMENTED** [review]
May be we can combine the AWS REGION and GCE Datacenter in pipeline code?

File: sdcm/utils/gce_utils.py [hunk 13]
```diff
@@ -28,13 +28,21 @@
 
 GceDriver = get_driver(Provider.GCE)  # pylint: disable=invalid-name
 
+# The keys are the region name, the value is the available zones, which will be used for random.choice()
+SUPPORT_REGIONS = {
```
Comment (bentsi) [inline sdcm/utils/gce_utils.py:32]: `SUPPORT_REGIONS` -> `SUPPORTED_REGIONS`

File: sdcm/utils/gce_utils.py (hunk 12 above)
Comment (bentsi) [reply sdcm/utils/gce_utils.py:45]: Releng will handle this, they have the same problem for AWS and they solved it by adding a list of supported regions for SCT tests

File: sdcm/utils/gce_utils.py [hunk 14]
```diff
@@ -28,13 +28,21 @@
...
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
-        # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
-        return f"{region}-{random.choice('bccdd')}"  # choose zones c and d twice as often as zone b
-    return region
+    for support_region, zone_string in SUPPORT_REGIONS.items():
+        if region == support_region:
+            if region.count("-") != 1:
```
Comment (bentsi) [inline sdcm/utils/gce_utils.py:42]: how this situation can happen? why are you checking this?

File: sdcm/utils/gce_utils.py (hunk 10 above)
Comment (bentsi) [reply sdcm/utils/gce_utils.py:41]: @dkropachev no, we should have only region part here, the function is called `append_zone` we append availability zone suffix to the region name

File: vars/getJenkinsLabels.groovy [hunk 15]
```diff
@@ -33,11 +35,29 @@ def call(String backend, String region=null) {
...
             println("Found AWS builder with label: " + label)
             return [ "label": label, "region": region ]
-        }
-        else{
+        } else {
             throw new Exception("=================== AWS region ${region} not supported ! ===================")
         }
 
+    }
+    else if (cloud_provider == 'gce' && datacenter)
```
Comment (bentsi) [inline vars/getJenkinsLabels.groovy:43]: Amos, this is a copy-paste!!!
You could solve this by adding simple if and generalizing cloud provider-related strings.
```
    if cloud_provider == 'aws': {
        def supported_regions = ["eu-west-2", "eu-north-1", "eu-central-1"]
        }
    else if  cloud_provider == 'gce':
         def supported_regions = ["us-east1", "us-west1"]
         }
     else {
         throw new Exception("Unsupported cloud provider")
         }
```

**Review by bentsi — COMMENTED** [review]
1) address comments
2) let's be consistent and call region as **region** and not data center

File: sdcm/utils/gce_utils.py [hunk 16]
```diff
@@ -28,13 +28,21 @@
...
 
+# The keys are the region name, the value is the available zones, which will be used for random.choice()
+SUPPORT_REGIONS = {
+    # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
+    # choose zones c and d twice as often as zone b
+    'us-east1': 'bccdd',
+    'us-west1': 'abc'}
+
 
 def append_zone(region: str) -> str:
```
Comment (dkropachev) [inline sdcm/utils/gce_utils.py:39]:
```suggestion
def append_zone(region: str) -> str:
    dash_count = region.count("-")
    if dash_count == 2:
        # return as is if region already has availability zone in it
        return region
    if dash_count != 1:
        raise Exception(f'Wrong region name: {region}')
    availability_zones = SUPPORT_REGIONS.get(region, None)
    if not availability_zones:
        raise Exception(f'Unsupported region: {region}')
    return f"{region}-{random.choice(availability_zones)}"
```

**Review by dkropachev — CHANGES_REQUESTED** [review]

**Comment by amoskong** [comment]
> May be we can combine the AWS REGION and GCE Datacenter in pipeline code?

The default aws_region and gce_datacenter are different, so we can't use one parameter.

File: vars/getJenkinsLabels.groovy (hunk 11 above)
Comment (amoskong) [reply vars/getJenkinsLabels.groovy:19]: They are needed to match the builders in special region.

In the past, we only used one gce region (us-east), and all gce builders have one label: gce-sct-builders.

File: vars/perfRegressionParallelPipeline.groovy [hunk 17]
```diff
@@ -24,6 +24,10 @@ def call(Map pipelineParams) {
                description: 'us-east-1|eu-west-1',
                name: 'aws_region')
 
+            string(defaultValue: "${pipelineParams.get('gce_datacenter', '')}",
```
Comment (dkropachev) [inline vars/perfRegressionParallelPipeline.groovy:27]: Should not we have default here too ?

File: vars/rollingUpgradePipeline.groovy [hunk 18]
```diff
@@ -22,6 +22,9 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('aws_region', 'eu-west-1')}",
                description: 'Supported: us-east-1|eu-west-1|eu-west-2|eu-north-1|random (randomly select region)',
                name: 'aws_region')
+            string(defaultValue: "${pipelineParams.get('gce_datacenter', '')}",
```
Comment (dkropachev) [inline vars/rollingUpgradePipeline.groovy:25]: Should not we have default here too ?

**Review by dkropachev — CHANGES_REQUESTED** [review]
@amoskong , good job, let's rebase and run provision tests over it

File: vars/perfRegressionParallelPipeline.groovy (hunk 17 above)
Comment (amoskong) [reply vars/perfRegressionParallelPipeline.groovy:27]: added a default us-east1.

Comment (amoskong) [reply vars/perfRegressionParallelPipeline.groovy:27]: added a default us-east1.

File: vars/rollingUpgradePipeline.groovy (hunk 18 above)
Comment (amoskong) [reply vars/rollingUpgradePipeline.groovy:25]: added a default us-east1.

File: vars/runCleanupResource.groovy [hunk 19]
```diff
@@ -38,7 +41,7 @@ def call(Map params, String region){
             ./docker/env/hydra.sh --execute-on-runner \${RUNNER_IP} clean-resources --post-behavior --test-id \$SCT_TEST_ID
         else
             echo "SCT runner IP file is empty. Probably SCT Runner was not created."
-            exit 1
+            ./docker/env/hydra.sh clean-resources --post-behavior --test-id \$SCT_TEST_ID
```
Comment (dkropachev) [inline vars/runCleanupResource.groovy:44]: @bentsi , are we going to support non-runner by longevity subroutines (`runSendEmail.groovy`, `runCollectLogs.groovy`, `runCleanupResource.groovy`) ?
If that is true we need to address it on other rooutines
if it is not true, we need to revert this change

File: vars/rollingUpgradePipeline.groovy [hunk 20]
```diff
@@ -86,14 +92,27 @@ def call(Map pipelineParams) {
...
+                                             "AWS_SECRET_ACCESS_KEY=${env.AWS_SECRET_ACCESS_KEY}",
+                                             "SCT_TEST_ID=${UUID.randomUUID().toString()}",]) {
 
-                                        stage("Upgrade from ${base_version}") {
+                                        stage("Create SCT Runner for ${base_version}") {
                                             catchError(stageResult: 'FAILURE') {
                                                 wrap([$class: 'BuildUser']) {
                                                     dir('scylla-cluster-tests') {
                                                         checkout scm
+                                                        timeout(time: 5, unit: 'MINUTES') {
```
Comment (dkropachev) [inline vars/rollingUpgradePipeline.groovy:103]: @bentsi , @amoskong  Is converting `rollingUpgradePipeline.groovy` to use runners part of the PR ?

File: vars/artifactsPipeline.groovy [hunk 21]
```diff
@@ -89,6 +92,21 @@ def call(Map pipelineParams) {
                     }
                 }
             }
+            stage('Create SCT Runner') {
```
Comment (dkropachev) [inline vars/artifactsPipeline.groovy:95]: This won't work unless you convert other steps to use longevity routines. Otherwise, runner will be just haning there and not going to be used

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: vars/runCleanupResource.groovy (hunk 19 above)
Comment (amoskong) [reply vars/runCleanupResource.groovy:44]: Currently, some jobs are not executed on runner (Gce backend), but it failed in cleaning the resource.
In this PR, I switched all (almost all) gce backend jobs to use sct runner, so it won't be a problem.

File: vars/rollingUpgradePipeline.groovy (hunk 20 above)
Comment (amoskong) [reply vars/rollingUpgradePipeline.groovy:103]: Yes.

Comment (amoskong) [reply vars/rollingUpgradePipeline.groovy:103]: The upgrade tests and artifacts-tests are the major part to use GCE backend.
We want to switch them to use (gce) sct runner.

File: vars/artifactsPipeline.groovy (hunk 21 above)
Comment (amoskong) [reply vars/artifactsPipeline.groovy:95]: Fixed. Thanks

File: vars/runCleanupResource.groovy (hunk 19 above)
Comment (bentsi) [reply vars/runCleanupResource.groovy:44]: @dkropachev yes, some jobs don't need SCT Runner, like artifact tests

**Comment by bentsi** [comment]
@dkropachev can you go over the comments you added and resolve them if they were addressed?

File: vars/runCleanupResource.groovy (hunk 19 above)
Comment (dkropachev) [reply vars/runCleanupResource.groovy:44]: @amoskong, @bentsi , We need to decide how do we check in the pipeline routine weather it is running on runner or not.
It used to be that we relay on a backend.
Now you are changing it, to relay on content of `sct_runner_ip`, which I'am totally okay with.
But we need to have this logic be the same across all routines, right now we are patching only one, which is not ok.

Also if we are going for `sct_runner_ip` content, you need to redo the logic to reflect that, ie. to have:

```
    echo "Starting to clean resources ..."
    RUNNER_IP=\$(cat sct_runner_ip||echo "")
    if [[ -n "\${RUNNER_IP}" ]] ; then
        ./docker/env/hydra.sh --execute-on-runner \${RUNNER_IP} clean-resources --post-behavior --test-id \$SCT_TEST_ID
    else
        ./docker/env/hydra.sh clean-resources --post-behavior --logdir "`pwd`"
    fi
    echo "Finished cleaning resources."

```

Instead of 

```
    echo "Starting to clean resources ..."
    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" ]]; then
        RUNNER_IP=\$(cat sct_runner_ip||echo "")
        if [[ -n "\${RUNNER_IP}" ]] ; then
            ./docker/env/hydra.sh --execute-on-runner \${RUNNER_IP} clean-resources --post-behavior --test-id \$SCT_TEST_ID
        else
            echo "SCT runner IP file is empty. Probably SCT Runner was not created."
            exit 1
        fi
    else
        ./docker/env/hydra.sh clean-resources --post-behavior --logdir "`pwd`"
    fi
    echo "Finished cleaning resources."
```

**Review by dkropachev — CHANGES_REQUESTED** [review]

**Comment by amoskong** [comment]
@dkropachev Thanks for your review.

I think it's fine to just check the sct_runner_ip content, and execute on the runner when the file contains a valid IP address.
Then we only need to check the backend in creating the runner.

**Review by dkropachev — APPROVED** [review]
Sorry, my bad, did not stop it right away.
If you look at how we call `createSctRunner`, it looks like this:

```
                    catchError(stageResult: 'FAILURE') {
                        script {
                            wrap([$class: 'BuildUser']) {
                                dir('scylla-cluster-tests') {
                                    timeout(time: 5, unit: 'MINUTES') {
                                        createSctRunner(params, runnerTimeout , builder.region)
                                    }
                                }
                            }
                        }
                    }
```

which means if it fails we continue to run pipeline, in this case it is going to be executed on the jenkins.

**Comment by dkropachev** [comment]
Provision failure is not related to the PR

**Comment by amoskong** [comment]
> Provision failure is not related to the PR

Yes, it failed for a truncate nemesis, the test table doesn't exist.
We can ignore the err.

**Comment by amoskong** [comment]
> If you look at how we call `createSctRunner`, it looks like this:
> 
> ```
>                     catchError(stageResult: 'FAILURE') {
>                         script {
>                             wrap([$class: 'BuildUser']) {
>                                 dir('scylla-cluster-tests') {
>                                     timeout(time: 5, unit: 'MINUTES') {
>                                         createSctRunner(params, runnerTimeout , builder.region)
>                                     }
>                                 }
>                             }
>                         }
>                     }
> ```
> 
> which means if it fails we continue to run pipeline, in this case it is going to be executed on the jenkins.
> Not sure if it is an issue, i will leave it here, just in case.

I understand your concern, let @roydahan @bentsi to make a decision.
Personally I can accept the job to run without SCT runner in this case.
The job will finally get a failure result, we can identify the problem.


> 
> If we need to fail the pipeline when runner is not created we need to do this:
> 
> ```
>                         script {
>                             wrap([$class: 'BuildUser']) {
>                                 dir('scylla-cluster-tests') {
>                                     timeout(time: 5, unit: 'MINUTES') {
>                                         createSctRunner(params, runnerTimeout , builder.region)
>                                     }
>                                 }
>                             }
>                         }
> ```

**Review by roydahan — APPROVED** [review]

**Comment by bentsi** [comment]
we shouldn't run artifact tests on SCT Runner(((

**Comment by amoskong** [comment]
> we shouldn't run artifact tests on SCT Runner(((

Fixed by https://github.com/scylladb/scylla-cluster-tests/pull/3978
Thanks.
